modules/end-user-guide/pages/automatic-token-injection.adoc

```diff
@@ -0,0 +1,21 @@
+:navtitle: Automatic {orch-name} token injection
+:keywords: user-guide, token, injection
+:page-aliases: .:automatic-token-injection, overview:automatic-token-injection
+
+[id="automatic-token-injection_{context}"]
+= Automatic {orch-name} token injection
+
+This section describes how to use the {orch-name} user token that is automatically injected into workspace containers.
```

Comment:
It would be cool if this sentence could provide information on what you can do with the user token.

Reply:
fair point, will rework it